src/rules/no-empty-images.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @fileoverview Rule to prevent empty images in Markdown.
+ * @author 루밀LuMir(lumirlumir)
+ */
+
+//-----------------------------------------------------------------------------
+// Type Definitions
+//-----------------------------------------------------------------------------
+
+/**
+ * @typedef {import("../types.ts").MarkdownRuleDefinition<{ RuleOptions: []; }>}
+ * NoEmptyImagesRuleDefinition
+ */
+
+//-----------------------------------------------------------------------------
+// Rule Definition
+//-----------------------------------------------------------------------------
+
+/** @type {NoEmptyImagesRuleDefinition} */
+export default {
+	meta: {
+		type: "problem",
+
+		docs: {
+			recommended: true,
+			description: "Disallow empty images",
+			url: "https://github.com/eslint/markdown/blob/main/docs/rules/no-empty-images.md",
+		},
+
+		messages: {
+			emptyImage: "Unexpected empty image found.",
```

Comment:
I think we need to be more descriptive here.
```suggestion
			emptyImage: "Unexpected empty image found, markdown image must have a valid URL.",
```

Reply:
Like I mentioned above, In the [`no-empty-link`](https://github.com/eslint/markdown/blob/main/src/rules/no-empty-links.js) rule, it says the following:

![image](https://github.com/user-attachments/assets/fb039144-d4ee-4bb0-90c8-1059bfebeba9)

Is it okay to leave it as it is, or should I update both rules?